com.unity.render-pipelines.high-definition/Documentation~/Upgrading-from-2020.1-to-2020.2.md

```diff
@@ -165,3 +165,8 @@ With:
 ```
 protected override void Execute(CustomPassContext ctx) { ... }
 ```
+
+## Density Volume Mask Texture
+
+Previously, Density Mask Textures needed the __Density Volume Texture Tool__ in the __Window > Rendering__ menu to be converted from a 2D flipbook format to the 3D format required by the Density Volume Component.
```

Comment:
Previously, to convert a 2D flipbook texture to the 3D format that Density Mask Textures require, you needed to use the __Density Volume Texture Tool__ in the __Window > Rendering__ menu.